server/core/terraform/terraform_client.go

```diff
@@ -177,6 +189,7 @@ func NewClientWithDefaultVersion(
 		binDir:                  binDir,
 		downloader:              tfDownloader,
 		downloadBaseURL:         tfDownloadURL,
+		downloadsAllowed:        allowDownloads,
```

Comment:
Can we make this consistent with the other key/value?

```suggestion
		downloadAllowed:        tfDownloadAllowed,
```

Reply:
Addressed in https://github.com/runatlantis/atlantis/pull/2843/commits/588cc9109d87be7c3c8a673c74efc9ff266349b4